EricGray/cardsFactory.js

```diff
@@ -0,0 +1,184 @@
+console.log("Beginning of Question 3, card constructor redux")
```

Comment:
Nice work overall. Let me know if you want me to provide any commentary on the memory game section.